### PR TITLE
Update set systemlocale docs

### PR DESCRIPTION
**Pull request by erik0686**
The Set-WinSystemLocale powershell cmdlet allows to change the system locale, but this doesn't mean that the Windows Display Language needs to be changed as well, which can cause problems and configurations where font fallback doesn't work or the UI is not rendered properly. 

For this reason, I am adding a note on the documentation for people who might need this knowledge or are having problems resulting from this scenarios.

### CODE REVIEW

**Review by axelandrejs — APPROVED** [review]

File: docset/windows/international/Set-WinSystemLocale.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ For more information about the CultureInfo object, see [CultureInfo Class](http:
 ## OUTPUTS
 
 ## NOTES
+Please note that the system locale is primarily used for legacy codepage selection and font fallback. If the system locale is changed, it is highly recommended that the Windows Display Language is changed as well to match the locale, since the UI language may depend on the codepage and font to render properly. Failure to do so could result in configurations where the settings don't work as intended.
```
Comment (jefgen) [inline docset/windows/international/Set-WinSystemLocale.md:79]: Thanks for creating this PR!

I think I might suggest rewording this a bit. Perhaps something like the following:

> Please note that the System Locale setting on the computer is primarily used for legacy code page selection and font fallback. This setting is primarily used by applications that do not support Unicode. It should not be viewed from the sense of a traditional locale or culture info, and should not be confused with the user locale (Regional Format) setting.

> When changing the System Locale setting it is highly recommended to also change the Windows Display Language or UI language setting for the computer so that it matches the System Locale. In some cases, the UI language may depend on the code page and/or the font in order to render properly. Failure to do so can result in configurations where non-Unicode applications don't work as intended.

What do you think?

**Review by jefgen — CHANGES_REQUESTED** [review]

File: docset/windows/international/Set-WinSystemLocale.md (hunk 1 above)
Comment (erik0686) [reply docset/windows/international/Set-WinSystemLocale.md:79]: Sounds good Jeff :D Thanks!

**Review by jefgen — APPROVED** [review]
LGTM, thanks for adding this @erik0686!

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/international/Set-WinSystemLocale.md [hunk 2]
```diff
@@ -76,6 +76,9 @@ For more information about the CultureInfo object, see [CultureInfo Class](http:
 ## OUTPUTS
 
 ## NOTES
+Please note that the System Locale setting on the computer is primarily used for legacy code page selection and font fallback. This setting is primarily used by applications that do not support Unicode. It should not be viewed from the sense of a traditional locale or culture info, and should not be confused with the user locale (Regional Format) setting.
+
+When changing the System Locale setting it is highly recommended to also change the Windows Display Language or UI language setting for the computer so that it matches the System Locale. In some cases, the UI language may depend on the code page and/or the font in order to render properly. Failure to do so can result in configurations where non-Unicode applications don't work as intended.
```
Comment (dariomws) [inline docset/windows/international/Set-WinSystemLocale.md:81]:
```suggestion
Please note that the System Locale setting on the computer is primarily used for legacy code page selection and font fallback. This setting is primarily used by applications that do not support Unicode. It should not be viewed from the sense of a traditional locale or culture info, and should not be confused with the user locale (Regional Format) setting.

When changing the System Locale setting, it is highly recommended to also change the Windows Display Language or UI language setting for the computer so that it matches the System Locale. In some cases, the UI language may depend on the code page and/or the font to render properly. Failure to do so can result in configurations where non-Unicode applications don't work as intended.
```

**Review by dariomws — APPROVED** [review]
Just a very small suggestion.

**Review by get-itips — APPROVED** [review]

**Comment by dariomws** [comment]
Maybe the same information should be added to 2012 and 2012 R2 articles.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @dariomws's suggestions.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]
